### PR TITLE
RS-209: Access Control guide

### PR DESCRIPTION
**Pull request by atimin**
* Concepts
* Permissions
* Token management

### CODE REVIEW

File: docs/guides/access-control.mdx [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    </TabItem>
+    <TabItem value="Provisioning">
+        <CodeBlock language="yaml">
+            {CreateTokenProvisioned}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+
+:::info
```
Comment (AnthonyCvn) [inline docs/guides/access-control.mdx:135]: can be a warning

File: docs/guides/access-control.mdx [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+Each token has a set of permissions that define what actions can be performed with the token:
+
+- `full_access`: Allows the token to perform any action on the API.
+- `read`: List of buckets allowed to read data from entries.
+- `write`: List of buckets allowed to write data to entries or delete them.
+
+The table below shows the list of operations that can be performed with each permission:
+
+| Operation                | Unauthorized       | Authorized         | Read               | Write              | Full Access        |
```
Comment (AnthonyCvn) [inline docs/guides/access-control.mdx:42]: I don't get the meaning of Unauthorized and Authorized in the table

File: docs/guides/access-control.mdx [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+:::info
+If you don't need access control, you can disable token authentication by leaving `RS_API_TOKEN` unset.
+:::
+
+### Permissions and Access Control
+
+Each token has a set of permissions that define what actions can be performed with the token:
+
+- `full_access`: Allows the token to perform any action on the API.
+- `read`: List of buckets allowed to read data from entries.
```
Comment (AnthonyCvn) [inline docs/guides/access-control.mdx:37]: Can you clarify the set of permissions? I didn't get the`read` and `write` when reading quickly. Maybe something like:

- full_access: Allows the token to perform any action on the API without any restrictions.

- read: Allows the token to read data from specific buckets. The list of buckets that can be accessed for reading is defined when the token is created.

- write: Allows the token to write data to specific buckets, which includes adding new entries or deleting existing ones. The list of buckets that can be accessed for writing is defined when the token is created.

**Review by AnthonyCvn — CHANGES_REQUESTED** [review]
Looks great! Only the Permissions and Access Control section to clarify

File: docs/guides/access-control.mdx (hunk 1 above)
Comment (atimin) [reply docs/guides/access-control.mdx:135]: I keep it consistent with the other guides. Warning is for operations that can remove data.

**Pull request merged**